modules/core/src/lib/uniform-transition-manager.js

```diff
@@ -0,0 +1,67 @@
+import {lerp} from 'math.gl';
+import {normalizeTransitionSettings} from './attribute-transition-utils';
+
+export default class UniformTransitionManager {
+  constructor(timeline) {
+    this.transitions = new Map();
+    this.timeline = timeline;
+  }
+
+  get size() {
```

Comment:
Nit: Do we need this method? I only see this being used to check if `empty()` or rather `active()`, which migh be more self-documenting at call sites.